Plots/Satellites/NCL_sat_2.py

```diff
@@ -0,0 +1,516 @@
+"""
+NCL_sat_2.py
+===============
+This script illustrates the following concepts:
+   - Unpacking 'short' data
+   - Drawing filled contours over a satellite map
+   - Explicitly setting contour fill colors
+   - Finding local high pressure values
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/sat_2.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/sat_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import xarray as xr
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import numpy as np
+from sklearn.cluster import KMeans, DBSCAN
```

Comment:
```suggestion
from sklearn.cluster import DBSCAN
```
KMeans is unused in the code. Also, will this package need to be added to the conda_environment.yml?